modules/php/Game/Utils.php

```diff
@@ -13,4 +13,97 @@ public static function throwInvalidUserAction($msg)
   {
     throw new \BgaUserException($msg);
   }
+
+  public static function getActiveDoubleAgenda()
+  {
+    return 0 != self::getGame()->getGameStateValue("activeDoubleAgenda");
+  }
+
+  public static function getActiveInflation()
+  {
+    return 0 != self::getGame()->getGameStateValue("activeInflation");
+  }
+
+  public static function getActiveNoHandBonus()
+  {
+    return 0 != self::getGame()->getGameStateValue("activeNoHandBonus");
+  }
+
+  public static function getActivePartyBonus()
+  {
+    return 0 != self::getGame()->getGameStateValue("activePartyBonus");
+  }
+
+  public static function getActivePoorBonus()
+  {
+    return 0 != self::getGame()->getGameStateValue("activePoorBonus");
+  }
+
+  public static function getActiveRichBonus()
+  {
+    return 0 != self::getGame()->getGameStateValue("activeRichBonus");
+  }
+
+  public static function getActiveFirstPlayRandom()
+  {
+    return 0 != self::getGame()->getGameStateValue("activeFirstPlayRandom");
+  }
+
+  public static function isPartyInPlay()
+  {
+    $party_keeper_card_id = 16;
+    $party_keeper_card = self::getGame()->cards->getCard($party_keeper_card_id);
+    return $party_keeper_card["location"] == "keepers";
+  }
+
+  private static function getAllPlayersKeeperCount()
```

Comment:
There is a function directly in the Deck library for this: countCardsByLocationArgs
(see https://en.doc.boardgamearena.com/Deck).
I'll propose a simplification.

Reply:
Well.. I withdraw my comment, as the countCardsByLocationArgs function doesn't return anything if a player has no keeper...
I'll add a comment in the code for future-me :-)